SkietbaanBE/SkietbaanBE/Controllers/NotificationController.cs

```diff
@@ -13,10 +14,34 @@ namespace SkietbaanBE.Controllers
     public class NotificationController : Controller
     {
         private ModelsContext _context;
+        private NotificationMessages _notificationMessage;
 
-        public NotificationController(ModelsContext context)
+        public NotificationController(ModelsContext context, NotificationMessages notificationMessage)
         {
             _context = context;
+            _notificationMessage = notificationMessage;
+        }
+
+        [HttpPost]
+        public NotificationMessages AddNotification(string token)
+        {
+            var _document = new DocumentsController(_context);
+            var doccieLOS = _document.UserLOS(token); 
+            var doccieLOGS = _document.UserLOGS(token);
+            if (doccieLOS == "Document")
+            {
+                if(doccieLOGS == "Document")
+                {
+                    _notificationMessage.DocumenstNotification(token);
+                }
+            }else if (doccieLOGS == "Document")
```

Comment:
fix your if statement indentation, also look at your first and last if statement condition, they are the same but different return type, or maybe because of the indentation they look like this